src/studio-home/tabs-section/libraries-v2-tab/index.jsx

```diff
@@ -18,18 +26,26 @@ const LibrariesV2Tab = ({
   const intl = useIntl();
 
   const [currentPage, setCurrentPage] = useState(1);
+  const [isFiltered, setIsFiltered] = useState(false);
```

Comment:
IMHO, it's bad practice to have a state that can be trivially derived from other state. Because it can get out of sync and cause bugs, if you're not careful. It would be much better to have a line below like:

```
const isFiltered = Object.keys(filterParams).length > 0;
```

Reply:
Great point, I've updated to rely on the existing state rather than having a separate one just for determining if filtered or not: https://github.com/openedx/frontend-app-course-authoring/pull/1117/commits/a1d8feb6d42460967f9a9ff7e8ab83c826ce1545